### ISSUE TITLE
Jetpack.com: screen readers announce bullets of features lists

### ISSUE THREAD

**Issue by jkguidaven** [Automattic/jetpack#29839]
## What's the issue
On the Jetpack.com homepage, screen readers say the word "bullet" in English when reading the list of features of a product. They shouldn't read the marker of list items.

## How to reproduce the issue
- Visit https://jetpack.com/
- Scroll to the pricing section
- Open a screen reader of your choice and read the features of a product (see screencast below)

## Expected behavior
Screen readers should not announce the item marker. Try with other lists in the page.

## Captures
https://user-images.githubusercontent.com/1620183/227957402-6e5b7a76-f080-4096-a946-fb7179767d00.mov

**Comment by jkguidaven** [comment]
Hey @monsieur-z,

It looks like announcing "bullet" is the correct accessibility behavior.
I've tried looking for a solution to make the voiceover ignore the bullet, but I stumbled upon this [article](https://www.tempertemper.net/blog/accessibility-issues-when-removing-list-markers) indirectly suggesting it is the correct behavior. wdyt?

**Comment by monsieur-z** [comment]
Just reposting my answer here for the record.

> I'm not sure what to make of this article. With list-style-type: none;, VoiceOver still announces lists as such, and read items like ..., 1 of X. Maybe that was an issue some years ago?
> 
> To go back to the issue, it seems that even when we remove the Jetpack specific styles and leave only the user agent styles, VoiceOver does read the bullets. So I'd guess that's expected behaviour. I believe we can safely discard the issue then.
